test/orm/test_deferred.py

```diff
@@ -534,8 +534,10 @@ def go():
         q2 = q.options(undefer(Order.user_id))
         with expect_raises_message(
             sa.exc.InvalidRequestError,
-            r"Loader strategies for ORM Path\[Mapper\[Order\(orders\)\] -> "
-            r"Order.user_id\] conflict",
+            r"Loader strategy replacement "
+            r"ORM Path\[Mapper\[Order\(orders\)\] -> Order.user_id\] "
+            r"is in conflict with existing strategy "
+            r"ORM Path\[Mapper\[Order\(orders\)\] -> Order.user_id\]",
```

Comment:
ok this doesn't seem to add much value, since the two sides are the same.

@zzzeek do we have also a way of getting what option is used in each part of the path?

Reply:
I thought bringing in `_LoadElement.strategy` might add more value to this, but I realised that it is also equal in a lot of cases.

Reply:
im not really sure, I haven't looked at this code in a long time.  I do know it's vastly simpler code than it was in 1.4 and earlier